spark/dl/src/main/scala/com/intel/analytics/bigdl/python/api/PythonBigDL.scala

```diff
@@ -1531,6 +1531,13 @@ class PythonBigDL[T: ClassTag](implicit ev: TensorNumeric[T]) extends Serializab
       numInputDims)
   }
 
+  def createMultipleDimUnsqueeze(pos: Array[Int],
```

Comment:
I don't think we need 2 separate bridge method here, please consolidate it into one,  removing above one and use this one works for both.

Reply:
Ok, I have change it into one method.